packages/coding-standard/docs/phpstan_rules.md

```diff
@@ -2598,6 +2598,49 @@ class SomeClass
 
 <br>
 
+## Defined New Argument should be Always Constant Value
+
+- class: [`RequireNewArgumentConstantRule`](../src/Rules/RequireNewArgumentConstantRule.php)
+
+```yaml
+# phpstan.neon
+services:
+    -
+        class: Symplify\CodingStandard\Rules\RequireNewArgumentConstantRule
+        tags: [phpstan.rules.rule]
+        arguments:
+            constantArgByNewByType:
+                Symfony\Component\Console\Input\InputOption: [2]
+```
+
+```php
+<?php
+
+
+declare(strict_types=1);
+
+use Symfony\Component\Console\Input\InputOption;
+
+new InputOption('config', 'c', 2, 'Config file');
```

Comment:
Looks good :+1: 

Could you also add position for 'debug'?

I guess this is the configuration:

```yaml
...
             constantArgByNewByType:
                Symfony\Component\Console\Input\InputOption: [0, 2]
```

Reply:
@TomasVotruba implemented in documentation, do you want position `[0, 2]` to be added to `symplify-rules.neon` as well ?

Reply:
I've applied position [0, 2] to symplify-rules.neon as well

Reply:
I've fixed existing codebase new phpstan error message

Reply:
Thank you